bindings/go/evmc/evmc.go

```diff
@@ -478,6 +487,13 @@ func evmcBytes32(in Hash) C.evmc_bytes32 {
 	return out
 }
 
+func evmcBytes32_ptr(in []Hash) *C.evmc_bytes32 {
```

Comment:
Since it is only used in the `host.go` file, this function should be moved there.